csv/csv_parse_stream_test.ts

```diff
@@ -351,9 +349,7 @@ x,,,
 });
 
 function createReadableStreamFromString(s: string): ReadableStream<string> {
-  return readableStreamFromReader(new StringReader(s)).pipeThrough(
-    new TextDecoderStream(),
-  );
+  return ReadableStream.from(s);
 }
```

Comment:
We can remove the use of `createReadableStreamFromString()` and just use `ReadableStream.from()` instead.